src/prefect/projects/steps/core.py

```diff
@@ -47,22 +59,23 @@ def _get_function_for_step(fully_qualified_name: str, requires: Optional[str] =
     return step_func
 
 
-async def run_step(step: dict) -> dict:
+async def run_step(step: Dict, upstream_outputs: Optional[Dict] = None) -> Dict:
     """
     Runs a step, returns the step's output.
 
     Steps are assumed to be in the format `{"importable.func.name": {"kwarg1": "value1", ...}}`.
 
-    The 'requires' keyword is reserved for specific purposes and will be removed from the
+    The 'id and 'requires' keywords is reserved for specific purposes and will be removed from the
     inputs before passing to the step function:
 
     This keyword is used to specify packages that should be installed before running the step.
     """
-    fqn, inputs = step.popitem()
+    fqn, inputs = _get_step_fully_qualified_name_and_inputs(step)
```

Comment:
Do you think this fn needs a test since it's a critical path or no?

Reply:
`_get_step_fully_qualified_name_and_inputs` is very small and exists to prevent duplication. I think it'll get enough coverage through the `run_step` and `run_steps` tests.

Reply:
👍